### PR TITLE
chore: conformance report for KIC 3.1.1

### PR DESCRIPTION
**Pull request by mlavacca**
**What type of PR is this?**

/area conformance

**What this PR does / why we need it**:

Conformance report for Kong ingress Controller 3.1.1

**Which issue(s) this PR fixes**:

Fixes #

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Review by shaneutt — APPROVED** [review]
/ok-to-test
/approve
/cc @robscott @youngnick

File: conformance/reports/v1.0.0/kong-kubernetes-ingress-controller/README.md [hunk 1]
```diff
@@ -5,5 +5,6 @@
 |API channel|Implementation version|Mode|Report|
 |-----------|----------------------|----|------|
 |x|[v3.0.2](https://github.com/Kong/kubernetes-ingress-controller/releases/tag/v3.0.2)|x|[link](./v3.0.2-report.yaml)|
+|x|[v3.1.1](https://github.com/Kong/kubernetes-ingress-controller/releases/tag/v3.1.1)|x|[link](./v3.1.1-report.yaml)|
 
 ## Reproduce
```
Comment (robscott) [inline conformance/reports/v1.0.0/kong-kubernetes-ingress-controller/README.md:10]: Are we only going to start requiring this after the v1.1 release or should we just require this for all future conformance reports submitted, including this one?

Comment (shaneutt) [reply conformance/reports/v1.0.0/kong-kubernetes-ingress-controller/README.md:10]: I don't see a big downside from just requiring this from here on out :thinking:

**Review by sunjayBhatia — APPROVED** [review]
looks good as far as i can tell 👍🏽

File: conformance/reports/v1.0.0/kong-kubernetes-ingress-controller/README.md (hunk 1 above)
Comment (mlavacca) [reply conformance/reports/v1.0.0/kong-kubernetes-ingress-controller/README.md:10]: Yes, I think it makes sense. I'll add the steps to reproduce 👍

Comment (mlavacca) [reply conformance/reports/v1.0.0/kong-kubernetes-ingress-controller/README.md:10]: I've added the "reproduce" section.

**Comment by robscott** [comment]
Thanks @mlavacca!

/lgtm

**Pull request merged**